webapp/cypress/component/NavbarTest.cy.jsx

```diff
@@ -116,7 +116,7 @@ describe("Navbar", () => {
     cy.url().should("include", "/starting-materials");
 
     cy.contains("Equipment").click();
-    cy.url().should("include", "/equipment");
+    cy.url().should("include", "/equipments");
```

Comment:
```suggestion
    cy.url().should("include", "/equipment");
```

I think the original is correct, though the conventions of pluralising API routes do get confusing for words that are self-plural...